runtime/acala/src/lib.rs

```diff
@@ -1469,6 +1469,33 @@ impl<I: From<Balance>> frame_support::traits::Get<I> for TxFeePerGasV2 {
 	}
 }
 
+pub struct RandomnessSource<T>(sp_std::marker::PhantomData<T>);
```

Comment:
we should be able to make this reusable across runtime by having `ParachainSystem` to be passed as a generic parameter